### PR TITLE
[strings][categories]Add categories and en-ru translations for national parks, nature reserves, capes, glaciers, geysers

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

**Comment by mpimenov** [comment]
Tag as [strings], please.

File: data/categories.txt [hunk 1]
```diff
@@ -9265,3 +9265,23 @@ sponsored-holiday
...
+
+leisure-nature_reserve|@tourism
+en:Nature reserve
+ru:Заповедник
+
+natural-cape
+en:Cape
+ru:Мыс
+
+natural-geiser
```
Comment (mpimenov) [inline data/categories.txt]: s/geiser/geyser/g ?
https://wiki.openstreetmap.org/wiki/Tag:natural%3Dgeyser

Comment (tatiana-yan) [reply data/categories.txt]: fixed, спасибо

File: data/categories.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Category groups referenced in many categories, should be defined before being referenced.
```
Comment (mpimenov) [inline data/categories.txt]: Это разве не так?

**Pull request merged**